subtests/docker_cli/tag/tag.py

```diff
@@ -20,6 +20,7 @@
 from dockertest.images import DockerImage
```

Comment:
They always could have multiple tags, not sure I understand the commit message, but TBH I haven't played with 1.10 at all yet, so it's probably fine.